js_modules/dagster-ui/packages/ui-core/src/assets/AssetGroupRoot.tsx

```diff
@@ -135,10 +135,6 @@ export const ASSET_GROUP_METADATA_QUERY = gql`
       autoMaterializePolicy {
         policyType
       }
-      automationPolicySensor {
-        id
-        ...SensorSwitchFragment
-      }
```

Comment:
i took out the asset group stuff part for now because it could result in a ton of sensors for a large asset group, and even the current impl potentially has some performance issues unless we do more batching on the server

Reply:
👍